python/pyarrow/tests/test_fs.py

```diff
@@ -467,6 +501,11 @@ def skip_fsspec_s3fs(fs):
         pytest.xfail(reason="Not working with fsspec's s3fs")
 
 
+def skip_azure(fs, reason):
+    if fs.type_name == "abfs":
+        pytest.xfail(reason=reason)
```

Comment:
Can we use `pytest.skip` here?

Reply:
I can do. I just copied this from `skip_fsspec_s3fs` above.

Reply:
Done